### DISCUSSION TITLE
Question: Is possible use the hooks useDeno on api class?

### DISCUSSION THREAD

**Discussion by vincenzopalazzo** [alephjs/aleph.js#210] [Q&A]
I'm trying to use useDeno hooks in the API code but I start to understand that this is not possible.

I can ask why? or I'm missing somethings?

**Comment by ije** [comment]
@vincenzopalazzo you can use the `withDeno` hoc or create your own hoc wrapper.
fyi: https://github.com/alephjs/aleph.js/blob/master/framework/react/hoc.ts#L38

**Reply by vincenzopalazzo** [reply]
Oh amazing, thanks to point me out this link, I will take a look now!

p.s: there are any good first issue for the developers that want contribute in this project but don't know where to start?


Thanks!

**Reply by ije** [reply]
thanks a lot @vincenzopalazzo, this project is in very early stage, the code changes vary often, but you are welcome to contribute on it. you can read our contributing doc: https://github.com/alephjs/aleph.js/blob/master/CONTRIBUTING.md

**Reply by vincenzopalazzo** [reply]
In addition, Maybe I don't catch your suggestion, I want to use useDeno in the API class, but I receive the following error

```
INFO modify /lib/useNode.ts
ERROR invoke API: TypeError: The requested module '../../-/deno.land/x/aleph@v0.3.0-alpha.14/mod.js' does not provide an export named 'useDeno'
    at async Server.handle (https://deno.land/x/aleph@v0.3.0-alpha.14/server/server.ts:156:41)

```

My class is under path /api/lnndoe/node.tys

**Reply by ije** [reply]
sorry i misunderstood, in the API moudle, no need to use the `useDeno` hook since it runs in deno not browser, `useDeno` is a react hook to handle SSR data.

**Reply by vincenzopalazzo** [reply]
It is enought this call? `const getInfoNode = await (await fetch("http://localhost:7000/utility/getinfo")).json();`

**Reply by ije** [reply]
pls try~
